Java/libraries/recognizers-text-date-time/src/main/java/com/microsoft/recognizers/text/datetime/english/extractors/EnglishDurationExtractorConfiguration.java

```diff
@@ -0,0 +1,150 @@
+package com.microsoft.recognizers.text.datetime.english.extractors;
+
+import java.util.regex.Pattern;
+
+import com.google.common.collect.ImmutableMap;
+import com.microsoft.recognizers.text.IExtractor;
+import com.microsoft.recognizers.text.datetime.DateTimeOptions;
+import com.microsoft.recognizers.text.datetime.config.BaseOptionsConfiguration;
+import com.microsoft.recognizers.text.datetime.extractors.config.IDurationExtractorConfiguration;
+import com.microsoft.recognizers.text.datetime.resources.EnglishDateTime;
+import com.microsoft.recognizers.text.number.english.extractors.CardinalExtractor;
+import com.microsoft.recognizers.text.utilities.RegExpUtility;
+
+public class EnglishDurationExtractorConfiguration extends BaseOptionsConfiguration implements IDurationExtractorConfiguration {
+
+  public static final Pattern DurationUnitRegex = RegExpUtility.getSafeRegExp(EnglishDateTime.DurationUnitRegex, Pattern.CASE_INSENSITIVE);
```

Comment:
All the DateTime regexes no longer need to be CASE_INSENSITIVE. This saves quite a bit in string allocations and conversions. :)